tests/unittests/CoreFoundation/CFTimeZoneTests.mm

```diff
@@ -0,0 +1,227 @@
+                                                                  (__bridge CFLocaleRef)[NSLocale systemLocale]));
+    ASSERT_OBJCEQ(expectedProperties[@"CFTimeZoneCopyLocalizedName Generic"],
+                  (__bridge NSString*)CFTimeZoneCopyLocalizedName(tz,
+                                                                  kCFTimeZoneNameStyleGeneric,
+                                                                  (__bridge CFLocaleRef)[NSLocale systemLocale]));
+    ASSERT_OBJCEQ(expectedProperties[@"CFTimeZoneCopyLocalizedName ShortGeneric"],
+                  (__bridge NSString*)CFTimeZoneCopyLocalizedName(tz,
+                                                                  kCFTimeZoneNameStyleShortGeneric,
+                                                                  (__bridge CFLocaleRef)[NSLocale systemLocale]));
+
+    ASSERT_OBJCEQ(expectedProperties[@"CFTimeZoneGetSecondsFromGMT"], [NSNumber numberWithDouble:CFTimeZoneGetSecondsFromGMT(tz, 0)]);
+}
+
+TEST(CFTimeZone, Create) {
+    // WinObjC draws on a separate database
+    NSDictionary* expected = @{
+#ifdef WINOBJC
+        @"CFTimeZoneCopyAbbreviation" : @"EST",
+#else
+        @"CFTimeZoneCopyAbbreviation" : @"GMT-5",
+#endif
+        @"CFTimeZoneCopyAbbreviation" : @"EST",
+        @"CFTimeZoneGetName" : @"EST",
+        @"CFTimeZoneCopyLocalizedName Standard" : @"GMT-05:00",
+        @"CFTimeZoneCopyLocalizedName ShortStandard" : @"GMT-5",
+        @"CFTimeZoneCopyLocalizedName DaylightSaving" : @"GMT-05:00",
+        @"CFTimeZoneCopyLocalizedName ShortDaylightSaving" : @"GMT-5",
+        @"CFTimeZoneCopyLocalizedName Generic" : @"GMT-05:00",
+        @"CFTimeZoneCopyLocalizedName ShortGeneric" : @"GMT-5",
+        @"CFTimeZoneGetSecondsFromGMT" : @-18000.000000
+    };
+    assertCFTimeZoneProperties(CFTimeZoneCreate(kCFAllocatorDefault, CFSTR("EST"), nullptr), expected);
+
+    // WinObjC draws on a separate database
+    expected = @{
+#ifdef WINOBJC
+        @"CFTimeZoneCopyAbbreviation" : @"America/Argentina/Buenos_Aires",
+#else
+        @"CFTimeZoneCopyAbbreviation" : @"GMT-3",
+#endif
+        @"CFTimeZoneGetName" : @"America/Argentina/Buenos_Aires",
+        @"CFTimeZoneCopyLocalizedName Standard" : @"GMT-03:00",
+        @"CFTimeZoneCopyLocalizedName ShortStandard" : @"GMT-3",
+        @"CFTimeZoneCopyLocalizedName DaylightSaving" : @"GMT-03:00",
+        @"CFTimeZoneCopyLocalizedName ShortDaylightSaving" : @"GMT-3",
+        @"CFTimeZoneCopyLocalizedName Generic" : @"Buenos Aires",
+        @"CFTimeZoneCopyLocalizedName ShortGeneric" : @"Buenos Aires",
+        @"CFTimeZoneGetSecondsFromGMT" : @-10800.000000
+    };
+    assertCFTimeZoneProperties(CFTimeZoneCreate(kCFAllocatorDefault, CFSTR("America/Argentina/Buenos_Aires"), nullptr), expected);
+
+    // WinObjC draws on a separate database
+    expected = @{
+#ifdef WINOBJC
+        @"CFTimeZoneCopyAbbreviation" : @"US/Pacific",
+#else
+        @"CFTimeZoneCopyAbbreviation" : @"PST",
+#endif
```

Comment:
I'll poke him about it - the name being the same as the abbreviation seems concerning.

---
In reply to: [72732357](https://github.com/Microsoft/WinObjC/pull/701#discussion_r72732357) [](ancestors = 72732357)